### PR TITLE
[Merge After #108] [DOCS] PATH cheatsheet

### PR DESCRIPTION
**Pull request by commoddity**
Created first draft of PATH quickstart cheatsheet for running 0-First Relay.

### CODE REVIEW

File: envoy/scripts/copy_envoy_config.sh [hunk 1]
```diff
@@ -15,16 +15,27 @@ if [ -f "$ENVOY_CONFIG_PATH" ]; then
...
-
-# Prompt for AUTH_AUDIENCE
-read -p "Enter AUTH_AUDIENCE (eg. 'https://auth.example.com/oauth/token'): " AUTH_AUDIENCE
-
-# Substitute sensitive variables manually using bash parameter expansion
-sed -e "s|\${AUTH_DOMAIN}|$AUTH_DOMAIN|g" \
-    -e "s|\${AUTH_AUDIENCE}|$AUTH_AUDIENCE|g" \
-    "$SCRIPT_DIR/../envoy.template.yaml" > "$ENVOY_CONFIG_PATH"
+# Prompt the user if they wish to use an OAuth provider
+read -p "Do you wish to use an OAuth provider to enable authorizing requests using JWT? (yes/no): " USE_OAUTH
```
Comment (Olshansk) [inline envoy/scripts/copy_envoy_config.sh:19]: @commoddity Can you make `no` the default please?

File: docusaurus/docs/develop/path/cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
+---
```
Comment (Olshansk) [inline docusaurus/docs/develop/path/cheatsheet.md:1]: Wdyt of adding a gif/video to this page as well?

File: docusaurus/docs/develop/path/cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+:::
+
+Once the `PATH Gateway` container is ready, you may send a relay to test.
+
+## 4. Send a Relay
+
+Check that the `PATH Gateway` is serving relays by running the following command yourself:
+
+```bash
+curl http://eth.localhost:3001/v1/endpoint_1 \
```
Comment (Olshansk) [inline docusaurus/docs/develop/path/cheatsheet.md:215]: Can you show an example to endpoint_3 (no auth) as well?

File: docusaurus/docs/develop/path/cheatsheet.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+    -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: api_key_1" \
+    -d '{"jsonrpc": "2.0", "id": 1, "method": "eth_blockNumber" }'
+```
+
+:::tip
+
+The `GatewayEndpoint` ID `endpoint_1` and API Key `api_key_1` are defined in the `local/path/envoy/.gateway-endpoints.yaml` file.
```
Comment (Olshansk) [inline docusaurus/docs/develop/path/cheatsheet.md:224]: Can we rename `endpoint_XXX` to be more semantic?

Left similar comments elsewhere

**Review by Olshansk — CHANGES_REQUESTED** [review]
@commoddity I was reviewing this on the side while resolving some fixes in prod, so I apologize if my comments are terse.

tl;dr I got it working too!

1. Double check that what I pushed works
- Run it locally again please
- Run docs locally too

2. See my comments

3. Shold be g2g after!

File: envoy/scripts/copy_envoy_config.sh (hunk 1 above)
Comment (commoddity) [reply envoy/scripts/copy_envoy_config.sh:19]: There's no default; it requires the user to answer `yes|y`or `no|n`.

Comment (commoddity) [reply envoy/scripts/copy_envoy_config.sh]: > @commoddity Can you make `no` the default please?

I've improved the prompt to read better; it now takes 1. and 2. as options:

```bash
Configure JWT Authentication:
1. Enable JWT Auth (requires an OAuth provider like Auth0, along with domain and audience details)
2. Disable JWT Auth
Select an option (1 or 2): 
> 
```

I've also updated the script to use `yq` to remove the JWT-specific parts of the `.envoy.yaml` file if the user does not wish to use JWT as those parts will interfere with the proper functioning of the Envoy proxy unless the user has an 0Auth provider configured. (This was a TODO that I figured I should just do now.)

File: docusaurus/docs/develop/path/cheatsheet.md (hunk 2 above)
Comment (commoddity) [reply docusaurus/docs/develop/path/cheatsheet.md:1]: Sure,  I can work on that later today.

File: docusaurus/docs/develop/path/cheatsheet.md (hunk 4 above)
Comment (commoddity) [reply docusaurus/docs/develop/path/cheatsheet.md]: Done.

**Review by Olshansk — APPROVED** [review]
Just finished going through the isntructions.

1. Updated
2. Reviewed
3. Validated

Let's merge #108 to main, merge this with main, and then get this one in!
